packages/flutterfire_ui/lib/src/i10n/lang/nl.dart

```diff
@@ -0,0 +1,263 @@
+  @override
+  final String forgotPasswordButtonLabel;
+  @override
+  final String forgotPasswordViewTitle;
+  @override
+  final String resetPasswordButtonLabel;
+  @override
+  final String verifyItsYouText;
+  @override
+  final String differentMethodsSignInTitleText;
+  @override
+  final String findProviderForEmailTitleText;
+  @override
+  final String continueText;
+  @override
+  final String countryCode;
+  @override
+  final String codeRequiredErrorText;
+  @override
+  final String invalidCountryCode;
+  @override
+  final String chooseACountry;
+  @override
+  final String enableMoreSignInMethods;
+  @override
+  final String signInMethods;
+  @override
+  final String provideEmail;
+  @override
+  final String goBackButtonLabel;
+  @override
+  final String passwordResetEmailSentText;
+  @override
+  final String forgotPasswordHintText;
+  @override
+  final String emailLinkSignInButtonLabel;
+  @override
+  final String signInWithEmailLinkViewTitleText;
+  @override
+  final String signInWithEmailLinkSentText;
+  @override
+  final String sendLinkButtonLabel;
+  @override
+  final String arrayLabel;
+  @override
+  final String booleanLabel;
+  @override
+  final String mapLabel;
+  @override
+  final String nullLabel;
+  @override
+  final String numberLabel;
+  @override
+  final String stringLabel;
+  @override
+  final String typeLabel;
+  @override
+  final String valueLabel;
+  @override
+  final String cancelLabel;
+  @override
+  final String updateLabel;
+  @override
+  final String northInitialLabel;
+  @override
+  final String southInitialLabel;
+  @override
+  final String westInitialLabel;
+  @override
+  final String eastInitialLabel;
+  @override
+  final String timestampLabel;
+  @override
+  final String latitudeLabel;
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+  const NlLocalizations({
+    this.emailInputLabel = 'E-mail',
+    this.passwordInputLabel = 'Wachtwoord',
+    this.signInActionText = 'Login',
+    this.registerActionText = 'Registreer',
+    this.signInButtonText = 'Login',
+    this.registerButtonText = 'Registreer',
+    this.linkEmailButtonText = 'Verder',
+    this.signInWithPhoneButtonText = 'Login met telefoon',
+    this.signInWithGoogleButtonText = 'Login met Google',
+    this.signInWithAppleButtonText = 'Login met Apple',
+    this.signInWithTwitterButtonText = 'Login met Twitter',
+    this.signInWithFacebookButtonText = 'Login met Facebook',
+    this.phoneVerificationViewTitleText = 'Voer uw telefoonnummer in',
+    this.verifyPhoneNumberButtonText = 'Verder',
+    this.verifyCodeButtonText = 'Verifieer',
+    this.verifyingPhoneNumberViewTitle = 'Vul code van SMS in',
+    this.unknownError = 'Er is een onbekende fout voorgekomen',
+    this.smsAutoresolutionFailedError =
+    'Automatisch de SMS code verifieren is niet gelukt. Voer handmatig de code in.',
```

Comment:
This is easier to read.
```suggestion
    'Automatisch verifiëren van de SMS code is niet gelukt. Voer de code handmatig in.',
```